packages/a11y/src/AriaLabelProps.ts

```diff
@@ -55,3 +55,15 @@ export type AriaLabelPropsWithLabel =
        */
       label: ReactNode;
     } & Partial<AriaLabelProps>);
+
+/**
+ * Conditionally requires {@link AriaLabelProps}
+ * if `children` is not provided
+ */
+export type AriaLabelPropsWithChildren =
```

Comment:
hm where is this relevant?

Reply:
I'm using it in InputOption. It can be useful when we want the text read by VO to be whatever the text content (children) is, instead of insisting on a `label` prop. 

(previously `InputOption` without a `label` prop would result in a TS error, which IMO is not correct)

Reply:
Can you add the bit about VO in the doc above? The context is really helpful